packages/common-all/src/types/typesv2.ts

```diff
@@ -626,12 +641,14 @@ export enum CalendarViewMessageType {
   "messageDispatcherReady" = "messageDispatcherReady",
 }
 
-export enum NoteViewMessageType {
+export enum NoteViewMessageEnum {
```

Comment:
I don't think it's necessary to include the word 'enum' in the name of an enum.

Reply:
reason is because we also introduce `NoteViewMessageType` which is the union of `NoteViewMessageEnum`  and `DMessageType`

Reply:
Got it - let's consider renaming in the future. At face value, I'm not sure what `NoteViewMessageType` vs `NoteViewMessageEnum` means.  `DMessageType` too - let's try to make them more specific.